### ISSUE TITLE
Problem with "cannot perform 'pointerUp' action without performing 'pointerDown' first"

### ISSUE THREAD

**Issue by jatalahd** [appium/appium-uiautomator2-server#470]
My configuration:
* appium v2.0.0-beta.35
* AndroidUiautomator2Driver's BaseDriver version is 8.5.3
* io.appium.uiautomator2.server version 5.6.1
* Appium-Python-Client version 2.2.0

In pure Selenium, when testing Web pages, this is possible:

`from selenium import webdriver`
`webdriver.ActionChains(driver).click_and_hold(ptt_button).perform()`
`print ("Can do stuff between!!!")`
`webdriver.ActionChains(driver).release().perform()`

But when doing this in Appium:

`from appium import webdriver`
`from selenium.webdriver.common.action_chains import ActionChains`
`ActionChains(driver1).click_and_hold(ptt_button).perform()`
`print ("Can do stuff between!!!")`
`ActionChains(driver1).release().perform()`

I get the exception

Caused by: io.appium.uiautomator2.utils.w3c.ActionsParseException: You cannot perform 'pointerUp' action without performing 'pointerDown' first at 0ms in 'mouse' chain
	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.applyPointerActionToEventsMapping(ActionsTokenizer.java:407)
	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.tokenize(ActionsTokenizer.java:517)
	at io.appium.uiautomator2.handler.W3CActions.safeHandle(W3CActions.java:83)

Why the functionality is different between Selenium and Appium? Would it be possible to make the functionality to be the same?

We are testing a Push-To-Talk communication app and we would need to trigger a PTT press in first device, do stuff in in other device and finally end PTT press in the first device. Basically this works if we do another very short click_and_hold() before release(), but I don't see this as a very clean way of handling this. The main question in this approach is what happens to the first ongoing click_and_hold(), is it safely released by the second overlapping click_and_hold() ?

**Comment by mykola-mokhnach** [comment]
I've never investigated if such scenario is possible/supported using Android events API. Feel free to test it and update the current server implementation if that works.

**Comment by mykola-mokhnach** [comment]
I'd rather say this is an improvement request. It's just the downstream topic is not quite simple and it requires some more theoretical knowledge to be worked on. You could start with https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/actions.md

**Comment by jatalahd** [comment]
You have made a very good job with this uiautomator2-server. You had the support already for this functionality, the only problem was that consecutive PointerUp events were (unnecessarily) restricted by the ActionsTokenizer. 

I made my first ever pull request, hopefully you will take that into consideration. With that change it is now possible to do button down and button up in two separate ActionChains:

ActionChains(driver1).click_and_hold(ptt_button).perform()
print ("CAN DO STUFF!!")
ActionChains(driver1).release(ptt_button).perform()

Tested using Python.
